pkg/features/list.go

```diff
@@ -51,4 +51,7 @@ var (
 
 	// ResyncDisabled disables the resync behavior of the kubernetes listeners in sensor
 	ResyncDisabled = registerFeature("Disable the re-sync", "ROX_RESYNC_DISABLED", false)
+
+	// RoxSyslogextraFields enables user to add additional key value pairs in syslog alert notification in cef format
+	RoxSyslogextraFields = registerFeature("Enable extra fields for syslog integration", "ROX_SYSLOG_EXTRA_FIELDS", false)
```

Comment:
We also need to add this to the FE type check in 
`ui/apps/platform/src/types/featureFlag.ts`
after `line 13`